<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>AboutDlg</class>
 <widget class="QDialog" name="AboutDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>About Barracuda IM Client</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="IconLabel" name="lb_icon" >
       <property name="frameShape" >
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Plain</enum>
       </property>
       <property name="psiIconName" stdset="0" >
        <string>psi/cudafull</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_name" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>7</hsizetype>
         <vsizetype>5</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="textFormat" >
        <enum>Qt::RichText</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tw_tabs" >
     <widget class="QWidget" name="tab_about" >
      <attribute name="title" >
       <string>About</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="lb_about" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>7</hsizetype>
           <vsizetype>7</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>The Barracuda IM client designed for the Barracuda IM Firewall.&lt;br&gt;
&lt;br&gt;
Copyright (C) 2005-2010 Barracuda Networks, Inc.
</string>
         </property>
         <property name="textFormat" >
          <enum>Qt::RichText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="URLLabel" name="uRLLabel1" >
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
         <property name="url" >
          <string>http://www.barracuda.com/im</string>
         </property>
         <property name="title" >
          <string>http://www.barracuda.com/im</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lb_about2" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>7</hsizetype>
           <vsizetype>7</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>This software is based on the Psi XMPP/Jabber client.&lt;br&gt;
&lt;br&gt;
Copyright (C) 2001-2010 The Psi Team.
</string>
         </property>
         <property name="textFormat" >
          <enum>Qt::RichText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="URLLabel" name="uRLLabel2" >
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
         <property name="url" >
          <string>http://psi-im.org/</string>
         </property>
         <property name="title" >
          <string>http://psi-im.org/</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_authors" >
      <attribute name="title" >
       <string>Authors</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="PsiTextView" name="te_authors" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_thanks" >
      <attribute name="title" >
       <string>Thanks To</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="PsiTextView" name="te_thanks" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_translation" >
      <attribute name="title" >
       <string>Translation</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="PsiTextView" name="te_translation" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>5</hsizetype>
           <vsizetype>7</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>I. M. Anonymous &lt;note text="replace with your real name">&lt;br>
&amp;nbsp;&amp;nbsp;&lt;a href="mailto:noone@nowhere.com">nowhere@nowhere.com&lt;/a> &lt;note text="replace with your email">&lt;br>
&amp;nbsp;&amp;nbsp;&lt;a href="http://blah.com">http://blah.com&lt;/a> &lt;note text="write your site URL here">&lt;br>
&amp;nbsp;&amp;nbsp;Jabber: &lt;a href="jabber:blah@jabber.org">blah@jabber.org &lt;note text="write your JID here">&lt;/a>&lt;br>
&amp;nbsp;&amp;nbsp;Translator &lt;note text="write description for the work, you've done">&lt;br>
&lt;br>
&lt;note text="you can add as many people here as you want">

&lt;note text="and translate following text">
Join the translation team today! Go to &lt;a href="http://psi-im.org/forum/forum/3">http://psi.affinix.com/forums/index.php?act=SF&amp;f=3&lt;/a> for further details!
</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_license" >
      <attribute name="title" >
       <string>License</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QTextEdit" name="te_license" >
         <property name="readOnly" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>418</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="IconButton" name="pb_close" >
       <property name="text" >
        <string>&amp;Close</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>IconButton</class>
   <extends></extends>
   <header>iconbutton.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>IconLabel</class>
   <extends></extends>
   <header>iconlabel.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>PsiTextView</class>
   <extends></extends>
   <header>psitextview.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>URLLabel</class>
   <extends></extends>
   <header>urllabel.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pb_close</sender>
   <signal>clicked(bool)</signal>
   <receiver>AboutDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>436</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel" >
     <x>369</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
